lib/src/folding/parsers/abstract.dart

```diff
@@ -30,7 +30,7 @@ abstract class AbstractFoldableBlockParser {
   void parse({
     required Result highlighted,
     required Set<Object?> serviceCommentsSources,
-    required List<CodeLine> lines,
+    required CodeLines lines,
```

Comment:
Why this parameter has been changed? Later here uses only lines field of this object